### PR TITLE
Added the anchor parameter to markers

### LINKED ISSUE #16: Custom marker icon placement point
It seems like the marker icon is always placed by the center if you use a custom marker icon.  I know there's a way to control the relative placement point for both Google maps API and IOS, so can we get that exposed via this API?

### PR DESCRIPTION
**Pull request by janoschp**
I added the anchor parameter to the Marker class constructor. I need this parameter because I'm using a custom icon for my markers which have a different anchor than the default one.

Fixes #16

### CODE REVIEW

File: lib/src/marker.dart [hunk 1]
```diff
@@ -119,6 +120,13 @@ class Marker {
   /// 0.0 means fully transparent, 1.0 means fully opaque.
   final double alpha;
 
+  /// The icon image point that will be placed at the [position] of the marker.
+  ///
+  /// The image point is specified in normalized coordinates: An anchor of
+  /// (0.0, 0.0) means the top left corner of the image. An anchor
+  /// of (1.0, 1.0) means the bottom right corner of the image.
```
Comment (janoschpelzer) [inline lib/src/marker.dart:127]: Copied from https://github.com/flutter/plugins/blob/01800a04f09cf5e8ec010e3027e5b9809094ee0c/packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/marker.dart#L45

**Comment by janoschp** [comment]
@LuisThein will anybody process this?

**Comment by Doppelklick** [comment]
@LuisThein can you accept the commit?

**Comment by janoschp** [comment]
@LuisThein  ?

**Comment by LuisThein** [comment]
Hi, sorry for not being able to approve this earlier.
Just fyi you can always include your forked packages through git dependencies:
https://dart.dev/tools/pub/dependencies#git-packages

This will be merged and a new release of the package will follow soon.

**Pull request merged**